app/services/manage_braintree_donation.rb

```diff
@@ -0,0 +1,58 @@
+class ManageBraintreeDonation
```

Comment:
I'd like to see something in the naming convention here that makes it explicit that this class is for passing messages onto ActionKit. It's very similar to our naming convention for managing donations for our local uses.